cfme/containers/image.py

```diff
@@ -129,7 +131,7 @@ def compliant(self):
 
 
 class ImageAllView(ContainerObjectAllBaseView):
-    TITLE_TEXT = 'Container Images'
+    summary = Text('//h1[normalize-space(.) = "Container Images"]')
```

Comment:
looks a bit weird. I assume there should be easier xpath w/o text comparison. This text comparison would look better in is_displayed property.